gravitee-apim-console-webui/angular.json

```diff
@@ -103,7 +103,7 @@
             "styles": [
               "src/index.scss",
               {
-                "input": "node_modules/swagger-ui-dist/swagger-ui.css",
+                "input": "node_modules/swagger-ui/dist/swagger-ui.css",
```

Comment:
I'm wondering if this is still necessary?

Reply:
Yes. If you don't add that, your swagger document will not be formatted